README.md
<h1 align="center"><strong>The Modern HomeServer</strong></h1>
<p align="center"><a href="README.md">Tip: a wider desktop view of this page.</a></p>
<p align="center">
  <a href="Justification.md">Why A Selfhosted Homeserver?</a> |
  <a href="https://www.docker.com/resources/what-container">Intro to microservices</a> |
  <a href="Recommendations.md">Hardware Recommendations</a> |
  <a href="https://github.com/zilexa/Homeserver/blob/master/README.md#requirements">How To Get Started?</a>
</p>

***

<h3 align="center">&bull; <strong>A fast, very low-maintenance, energy efficient selfhosted cloud.</strong></h3>
<h3 align="center">&bull; <strong>Can be used for any selfhosted system (from home automation to password manager).</strong></h3>
<h3 align="center">&bull; <strong>Can also be deployed and run in the background on your HomePC/Workstation.</strong></h3>
<h3 align="center">&bull; <strong>Carefully selected hardware recommendations.</strong></h3>
<h3 align="center">&bull; <strong>Carefully selected Operating System and server tools.</strong></h3>
<h3 align="center">&bull; <strong>Preconfigured automatic nightly/monthly maintenance.</strong></h3>
<h3 align="center">&bull; <strong>Carefully selected services for file cloud, password manager, media downloads and more.</strong></h3>

<sub>Note: I had zero experience when I started and learned everything by googling, spending time on fora, reddit and in documentations and by hours and days of trial&error. I made lots of mistakes. Now, in case of disaster I will use the scripts in this repository myself to get up and running again. I am documenting this because I haven't found a single source online that provides _all necessary information_ to get up and running. Also, lot's of things have been carefully chosen after testing alternatives. You can save lots of time with this guide! :) [basic Linux commands](https://www.hostinger.com/tutorials/linux-commands).</sub>

## Features
* A file cloud using: _**FileRun**_ - Access and share your files from everywhere, sync your devices and enjoy your photo albums from any device, anywhere in the world. Fast and secure!
* A fantastic password manager: _**VaultWarden**_ - a.k.a Bitwarden, the best password manager available. 
* Your own Office Online/Google Docs: _**OnlyOffice**_ - integrates nicely with FileRun. 
* Your own browser sync: _**Firefox Sync Server**_ - your history, passwords, form fill info etc no longer shared with 3rd parties and easy to sync from laptop to phone.
* A highly secured reverse web proxy to be able to use all these features via your own domain like https://my.domain.cloud. 
* Your own ad-filtering or parental filtering dns server: _**AdGuard Home**_ - no more ads on any device within your network and even on your phone when you leave the house.
* Your own very fast VPN: _**Wireguard**_ - Your own VPN server and easy management of clients. To access services you do not want to expose publicly online.
* Your own paper scanning and organizing system _**Paperless-ng**_ - helps you to automatically archive your household administration and access documents easily.
* Your own media center server _**JellyFin**_ - Watch your series and movies on any smart TV or device. Also when you leave the home via VPN.
* Automatically follow and retrieve your favourite tv shows and movies _**Sonarr Radarr Bazarr QBittorrent**_ Because nobody can pay for every streaming service just to watch the few shows or movies they like. 
* Pretty dashboards to monitor your server _**Grafana, Prometheus**_ - Not necessary, but handy and with notifications. 
* Your own homepage to quickly access each service in a single webpage _**Organzir**_ See preview here. 

For more details per service, [see here](https://github.com/zilexa/Homeserver/blob/master/Applications-Overview.md). 

***

### Requirements
Regardless of your hardware (x86-64 or ARM, see [Hardware Recommendations](Recommendations.md), for the scripts in this guide to work, the following is required, otherwise you may need to change the install commands, build software, figure out how to configure email notifications etc: 

- Arch based OS
- AUR (Arch User Repository)
- Pamac to be able to install AUR packages
- Pacman to be able to install packages
- BTRFS filesystem - with or without swapfile (no swap partition)

**[Manjaro](https://manjaro.org/)** on x86-64 (Intel/AMD) is the recommended OS of choice and used for this guide. It does not matter which [desktop flavor](https://manjaro.org/download/) you choose. I use Gnome but power users might want to consider KDE instead. 

<sub>This guide used to be Ubuntu-based. All my laptops/PCs and my parents systems ran Ubuntu Budgie. After 2 years I switched to Manjaro (Gnome edition) and it is a delight!
A much better **out-of-the-box experience**, more **user-friendly** (also for setting up a server! This is reflected in my scripts, they are a lot smaller)  **lightweight**, **small footprint**, much better and up to date single source of **high quality documentation (Arch Wiki)** and **MUCH easier to install applications + keep up to date** than any Ubuntu system. Also, better out-of-the-box BTRFS support. For small, flexible homeservers and personal laptops I strongly believe BTRFS is the best filesystem. If you have more needs, look at XFS/ZFS. This guide will use BTRFS. </sub>

***

### Step 1 - Install Operating System & Essential Tools
#### Step 1A -  How to install Manjaro? 
How to prep a USB stick: see [Manjaro First Steps > Using a Live System](https://manjaro.org/support/firststeps/). Further Instructions [here](https://github.com/zilexa/manjaro-gnome-post-install#quick-guide).

_Note_
* Requirement: select _BTRFS_ during setup with _no swap_, this allows you to later enable swapfile with hibernate support or zRAM.
* Consider using my [Manjaro Gnome Post-Install script](https://github.com/zilexa/manjaro-gnome-post-install). For example, it will automatically configure RDP, which is very handy for your server (you will be required to set credentials during script execution). I use this script for all my laptops/PCs also for family and friends, regardless if it is a media/NAS server or just home PC. 

#### Step 1B. How to properly install Docker and essential tools?
_Turn your system into a modern server with 1 click with [prep_server.sh](https://github.com/zilexa/Homeserver/blob/master/prep-server.sh) to automatically or manually install essential tools, apply basic configuration + required stuff for optional docker services._

Read through the [prep_server.sh](https://github.com/zilexa/Homeserver/blob/master/prep-server.sh) and see what it does (everything has comments). 
The PREP_DOCKER.SH script is carefully created to install the tools required to run all aspects of your server (from docker to mainentance tasks). \
Official documentation is used to install tools correctly (Docker Rootless Mode) and all but 1 tools will be installed via the Manjaro or AUR repository --> they will be automatically updated by the OS official repositories (compare that to Ubuntu/Debian based systems). 

Download and install it via: 
```
cd Downloads && wget https://raw.githubusercontent.com/zilexa/Homeserver/master/prep-server.sh
bash prep-server.sh
```
_Note_
> - A subvolume for Docker will be created --> allows extremely easy daily or hourly backups and recovery.
> - Installs Docker in rootless mode for enhanced security. This reduces the attack serface of your server. 
> - Allows OS support to send emails (with minimal set of tools and configuration), several Docker containers and your maintenance tasks will need this.
> - Installs several other essential tools, essential for example for data migration, backups, maintenance.
> - Optional config files for a few services (will ask y/n before downloading).For example if you are going to use torrents, consider using the QBittorrent config file. The Organizr config might be nice and will save you lots of time building your own "Start" page.

### Step 2 - Filesystem configuration
[Prepare the drives](https://github.com/zilexa/Homeserver/tree/master/filesystem). Understand your goal, make sane choices for your drives and create a datapool. 

### Step 3 - Network configuration

docker/changelog.md
# CHANGELOG 2022-04-22
# ADDED: labels to every non-exposed container to allow access each service via its own local subdomain. Please add these domains to your AdGuard Home>Settings>DNS Rewrites
# ADDED: labels added to every container with a UI to populate a simple startpage (by flame container)
# ADDED: Added Flame, a nice startpage. access via IP:5005 or http://start.o, go to settings, enable docker to show all services with the flame labels. 
# REMOVED: Per-category docker bridge networks, since all containers with a UI require to be in the web-proxy network for subdomain access.
#
# CHANGELOG 2022-04-11
# UPDATED: Hotio images have a new source. 
# CHANGED: For all services, first item is container_name, second is image source.  
#
# CHANGELOG 2022-04-08
# UPDATED: Now works with all latest containers. 
# CHANGED: 'true' to true everywhere.
# CHANGED: CADDY-DOCKER-PROXY: renamed contaimer name:'caddy-proxy to 'web-proxy', its network (unchanged) has the same name. 
# CHANGED: CADDY-DOCKER-PROXY: latest version requires an env variable, this has been added.
# CHANGED: CADDY-DOCKER-PROXY: requires you to run "docker network create web-proxy" BEFORE running compose!
#  
# CHANGELOG 2021-05-15
# CHANGED: Bitwarden_rs has been renamed by the developer to Vaultwarden, renamed the image, volume path, env variable
#
# CHANGELOG 2021-05-01
# REMOVED: PULLIO notify labels. It cannot send email notifications, only Discord which I and most people I believe don't use. 
## KEPT: PULLIO update labels, for media/download related containers, to update all of them with 1 command (manually or scheduled). 
# CHANGED: Scrutiny, an attempt to disable its interval based SMART scan, so that this can be done via the NIGHTLY, script.
## REASON: It will spin up all drives just for the SMART scan. Better to do this when they are already spinning. 
# CLEANUP: Changed order of values, to have consistency for each and every container.
#
# CHANGELOG 2021-04-28
## REMOVED: Watchtower. Replaced with Pullio bash script. For a weekly update check, running a daemon like Watchtower makes no sense. 
## CHANGED: VPN-proxy back to Frankfurt, for some reason Swiss doesn't work, the server TLS certificate is outdated. 
## REPLACED IMAGE: QBittorrent image with the one from Hotio because it has unrar and doesn't create files with root-only access.
## REPLACED ALL MEDIA RELATED IMAGES: all images are now from Hotio.
## CHANGED: Scrutiny requires a yaml config file to recognise NVME drives. see my github for an example. 
## CHANGED: Ports of VPN-proxy to allow access to QBittorrent as it listens on a different port for the webUI.  
#
# CHANGELOG 2021-04-13
## ADDED Guacamole! Now you can open your browser and access the desktop of your server. The image works, still have to test Guacamole. 
## Removed Transmission as nobody seems to use it anymore, even though it had full Sonarr compatibility, it is riddled with config issues:
## temp folder will never be used with Sonarr/Radarr, making the filesystem optimisations (disabling CoW & seperate subvolume for incomplete dir)
## Completely useless, allowing fragmentation. Also, the complete dir is changed during downloading via Sonarr. Devs believe this is all normal behaviour.
## ADDED QBittorrent, not from Linuxserver as it is 360MB versus this 150MB image. 
## ToDO: Will look into using Hotio QFlood image instead, the UI is much nicer but seems to lack important settings to increase disk cache etc.
## REMOVED unneccessary port binds: 80 for Caddy is not used. Now used for Organizr to access server homepage via url (use your servers /etc/hosts file to set an url). 
## ToDO: Will look into registering https for LAN only, to access all services via https organizr url (not sure if it makes sense to go this far). 
## CHANGED: pia wireguard vpn to swiss instead of nine eyes Germany.
## NOTE: I stopped using Syncthing due to this issue (my laptop SSD filled up within 30min): https://forum.syncthing.net/t/problems-with-renamed-directory/16611

docker/config.php
<?php
return array(
	'configVersion' => '2.1.235',
	'BOOKMARK-enabled' => false,
	'CHAT-Auth-include' => '1',
	'CHAT-appID-include' => '',
	'CHAT-authKey-include' => '',
	'CHAT-cluster-include' => '',
	'CHAT-enabled' => false,
	'CHAT-messageLoadLimit' => '200',
	'CHAT-newMessageSound-include' => 'plugins/sounds/default/newmessage.mp3',
	'CHAT-newMessageSoundz-include' => 'plugins/sounds/default/newmessage.mp3',
	'CHAT-secret' => '',
	'CHAT-useSSL' => false,
	'CHAT-userRefreshTimeout' => '60000',
	'HEALTHCHECKS-Auth-include' => '1',
	'HEALTHCHECKS-PingURL' => 'https://hc-ping.com/',
	'HEALTHCHECKS-all-items' => '',
	'HEALTHCHECKS-enabled' => false,
	'HEALTHCHECKS-option2-include' => '',
	'INVITES-EmbyTemplate' => '',
	'INVITES-enabled' => false,
	'INVITES-plex-movies-labels' => '',
	'INVITES-plex-music-labels' => '',
	'INVITES-plex-tv-labels' => '',
	'INVITES-plexLibraries' => '',
	'INVITES-type-include' => 'plex',
	'PHPMAILER-debugTesting' => false,
	'PHPMAILER-domain' => '',
	'PHPMAILER-emailTemplateCustom-include-Four' => '',
	'PHPMAILER-emailTemplateCustom-include-FourName' => 'Template #4',
	'PHPMAILER-emailTemplateCustom-include-FourSubject' => '',
	'PHPMAILER-emailTemplateCustom-include-One' => '',
	'PHPMAILER-emailTemplateCustom-include-OneName' => 'Template #1',
	'PHPMAILER-emailTemplateCustom-include-OneSubject' => '',
	'PHPMAILER-emailTemplateCustom-include-Three' => '',
	'PHPMAILER-emailTemplateCustom-include-ThreeName' => 'Template #3',
	'PHPMAILER-emailTemplateCustom-include-ThreeSubject' => '',
	'PHPMAILER-emailTemplateCustom-include-Two' => '',
	'PHPMAILER-emailTemplateCustom-include-TwoName' => 'Template #2',
	'PHPMAILER-emailTemplateCustom-include-TwoSubject' => '',
	'PHPMAILER-emailTemplateInviteUser' => '
	<h2>Hey there {user}!</h2><br />
	Here is the invite code to join my cool media server: {inviteCode}<br/>
	Head over to my website and enter the code to join: {domain}<br />
	',
	'PHPMAILER-emailTemplateInviteUserSubject' => 'You have been invited to join my server',
	'PHPMAILER-emailTemplateRegisterUser' => '
	<h2>Hey there {user}!</h2><br />
	Welcome to my site.<br/>
	If you need anything, please let me know.<br />
	',
	'PHPMAILER-emailTemplateRegisterUserEnabled' => true,
	'PHPMAILER-emailTemplateRegisterUserSubject' => 'Thank you For Registering',
	'PHPMAILER-emailTemplateReset' => '
	<h2>Hey there {user}!</h2><br />
	Looks like you forgot your password.  Well, I got you...  Here is your new password: {password}<br />
	If you want to change it once you log in, you can.  Head over to my website: {domain}<br />
	',
	'PHPMAILER-emailTemplateResetSubject' => 'Password Reset',
	'PHPMAILER-enabled' => false,
	'PHPMAILER-logo' => 'https://raw.githubusercontent.com/causefx/Organizr/v2-develop/plugins/images/organizr/logo-wide.png',
	'PHPMAILER-smtpHost' => '',
	'PHPMAILER-smtpHostAuth' => true,
	'PHPMAILER-smtpHostPassword' => '',
	'PHPMAILER-smtpHostPort' => '',
	'PHPMAILER-smtpHostSenderEmail' => 'no-reply@Organizr.tld',
	'PHPMAILER-smtpHostSenderName' => 'Organizr',
	'PHPMAILER-smtpHostType' => 'tls',
	'PHPMAILER-smtpHostUsername' => '',
	'PHPMAILER-template' => 'default',
	'PHPMAILER-verifyCert' => true,
	'SPEEDTEST-Auth-include' => '1',
	'SPEEDTEST-enabled' => false,
	'SPEEDTEST-option2-include' => '',
	'accentColor' => '',
	'accentTextColor' => '',
	'adminPingRefresh' => '60000',
	'alternateHomepageHeaders' => false,
	'authBackend' => '',
	'authBackendDomain' => '',
	'authBackendHost' => '',
	'authBackendHostPrefix' => '',
	'authBackendHostSuffix' => '',
	'authBaseDN' => '',
	'authDebug' => false,
	'authProxyEnabled' => false,
	'authProxyHeaderName' => '',
	'authProxyWhitelist' => '',
	'authType' => 'internal',
	'branch' => 'v2-master',
	'breezometerToken' => 'd95ab607392d4fa5bf64bb26a5cb2a06',
	'buttonColor' => '',
	'buttonHoverColor' => '',
	'buttonTextColor' => '',
	'buttonTextHoverColor' => '',
	'cacheImageSize' => '2',
	'calendarDefault' => 'month',
	'calendarEnd' => '90',
	'calendarEndTrakt' => '14',
	'calendarFirstDay' => '1',
	'calendarLimit' => '1000',
	'calendarLocale' => 'nl',
	'calendarRefresh' => '60000',
	'calendarStart' => '90',
	'calendarStartTrakt' => '14',
	'calendarTimeFormat' => 'H:mm',
	'calendariCal' => '',
	'commit' => 'n/a',
	'couchpotatoToken' => '',
	'couchpotatoURL' => '',
	'customCss' => '.collapse {
    display:block;
}',
	'customForgotPassText' => '',
	'customForgotPasswordText' => '',
	'customHTMLone' => '',
	'customHTMLtwo' => '',
	'customJava' => '',
	'customThemeCss' => '',
	'customThemeJava' => '',
	'dbLocation' => '/config/www/organizr/',
	'dbName' => 'organizrdb.db',
	'debugAreaAuth' => '1',
	'debugErrors' => false,
	'delugeCombine' => false,
	'delugeHideCompleted' => false,
	'delugeHideSeeding' => false,
	'delugePassword' => '',
	'delugeURL' => '',
	'description' => 'Organizr - Accept no others',
	'disableRecoverPass' => false,
	'disableRecoverPassword' => false,
	'embyTabName' => '',
	'embyTabURL' => '',
	'embyToken' => '',
	'embyURL' => '',
	'enableLocalAddressForward' => false,
	'expandCategoriesByDefault' => false,
	'favIcon' => '',
	'gaTrackingID' => '',
	'githubMenuLink' => false,
	'headerColor' => '',
	'headerTextColor' => '',
	'healthChecksTags' => '',
	'healthChecksToken' => '',
	'healthChecksURL' => 'https://healthchecks.io/api/v1/checks/',
	'hideRegistration' => false,
	'homepageCalendarAuth' => '4',
	'homepageCalendarEnabled' => false,
	'homepageCouchpotatoAuth' => '1',
	'homepageCouchpotatoEnabled' => false,
	'homepageCustomHTMLoneAuth' => '1',
	'homepageCustomHTMLoneEnabled' => false,
	'homepageCustomHTMLtwoAuth' => '1',
	'homepageCustomHTMLtwoEnabled' => false,
	'homepageCustomStreamNames' => '',
	'homepageDelugeAuth' => '1',
	'homepageDelugeEnabled' => false,
	'homepageDownloadRefresh' => '60000',
	'homepageEmbyAuth' => '1',
	'homepageEmbyEnabled' => false,
	'homepageEmbyLink' => ' http://app.emby.media/#!/item/item.html?id={id}&serverId={serverId}',
	'homepageEmbyRecent' => true,
	'homepageEmbyRecentAuth' => '1',
	'homepageEmbyStreams' => true,
	'homepageEmbyStreamsAuth' => '1',
	'homepageHealthChecksAuth' => '1',
	'homepageHealthChecksEnabled' => false,
	'homepageHealthChecksRefresh' => '60000',
	'homepageJackettAuth' => '1',
	'homepageJackettBackholeDownload' => false,
	'homepageJackettEnabled' => true,
	'homepageJdownloaderAuth' => '1',
	'homepageJdownloaderEnabled' => false,
	'homepageJellyStreamsAuth' => '1',
	'homepageJellyfinAuth' => '1',
	'homepageJellyfinEnabled' => true,
	'homepageJellyfinInstead' => false,
	'homepageJellyfinLink' => 'http://hostname:port/jellyfin/web/index.html#!/details?id={id}&serverId={serverId}',
	'homepageJellyfinRecent' => true,
	'homepageJellyfinRecentAuth' => '1',
	'homepageJellyfinStreams' => true,
	'homepageLidarrAuth' => '1',
	'homepageLidarrEnabled' => false,
	'homepageMonitorrAuth' => '1',
	'homepageMonitorrEnabled' => false,
	'homepageMonitorrRefresh' => '60000',
	'homepageNetdataAuth' => '1',
	'homepageNetdataEnabled' => true,
	'homepageNetdataRefresh' => '10000',
	'homepageNzbgetAuth' => '1',
	'homepageNzbgetEnabled' => false,
	'homepageOctoprintAuth' => '1',
	'homepageOctoprintEnabled' => false,
	'homepageOctoprintRefresh' => 10000,
	'homepageOmbiAuth' => '1',
	'homepageOmbiEnabled' => false,
	'homepageOmbiRequestAuth' => '1',
	'homepageOrderJackett' => '7',
	'homepageOrderMonitorr' => '27',
	'homepageOrderNetdata' => '1',
	'homepageOrderOctoprint' => '31',
	'homepageOrderPihole' => '11',
	'homepageOrderRadarrQueue' => '30',
	'homepageOrderSonarrQueue' => '4',
	'homepageOrderSpeedtest' => '29',
	'homepageOrderWeatherAndAir' => '28',
	'homepageOrdercalendar' => '6',
	'homepageOrdercustomhtml' => '12',
	'homepageOrdercustomhtmlTwo' => '13',
	'homepageOrderdeluge' => '16',
	'homepageOrderdownloader' => '24',
	'homepageOrderembynowplaying' => '14',
	'homepageOrderembyrecent' => '15',
	'homepageOrderhealthchecks' => '9',
	'homepageOrderjdownloader' => '25',
	'homepageOrderjellyfinnowplaying' => '2',
	'homepageOrderjellyfinrecent' => '3',
	'homepageOrdernzbget' => '17',
	'homepageOrderombi' => '22',
	'homepageOrderplexnowplaying' => '19',
	'homepageOrderplexplaylist' => '21',
	'homepageOrderplexrecent' => '20',
	'homepageOrderqBittorrent' => '5',
	'homepageOrderrTorrent' => '23',
	'homepageOrdersabnzbd' => '18',
	'homepageOrdertautulli' => '26',
	'homepageOrdertransmission' => '9',
	'homepageOrderunifi' => '8',
	'homepagePiholeAuth' => '1',
	'homepagePiholeCombine' => true,
	'homepagePiholeEnabled' => false,
	'homepagePiholeRefresh' => '10000',
	'homepagePlexAuth' => '1',
	'homepagePlexEnabled' => false,
	'homepagePlexPlaylist' => false,
	'homepagePlexPlaylistAuth' => '1',
	'homepagePlexRecent' => false,
	'homepagePlexRecentAuth' => '1',
	'homepagePlexRecentlyAddedMethod' => 'legacy',
	'homepagePlexStreams' => false,
	'homepagePlexStreamsAuth' => '1',
	'homepagePlexStreamsExclude' => '',
	'homepageRadarrAuth' => '1',
	'homepageRadarrEnabled' => true,
	'homepageRadarrQueueAuth' => '1',
	'homepageRadarrQueueCombine' => false,
	'homepageRadarrQueueEnabled' => false,
	'homepageRadarrQueueRefresh' => '60000',
	'homepageRecentLimit' => '50',
	'homepageRecentRefresh' => '60000',
	'homepageSabnzbdAuth' => '1',
	'homepageSabnzbdEnabled' => false,
	'homepageShowStreamNames' => true,
	'homepageShowStreamNamesAuth' => '1',
	'homepageSickrageAuth' => '1',
	'homepageSickrageEnabled' => false,
	'homepageSonarrAuth' => '1',
	'homepageSonarrEnabled' => true,
	'homepageSonarrQueueAuth' => '1',
	'homepageSonarrQueueCombine' => true,
	'homepageSonarrQueueEnabled' => true,
	'homepageSonarrQueueRefresh' => '60000',
	'homepageSpeedtestAuth' => '1',
	'homepageSpeedtestEnabled' => false,
	'homepageSpeedtestRefresh' => '1800000',
	'homepageStreamRefresh' => '60000',
	'homepageTautulliAuth' => '1',
	'homepageTautulliEnabled' => false,
	'homepageTautulliLibraryAuth' => '1',
	'homepageTautulliMiscAuth' => '1',
	'homepageTautulliRefresh' => '60000',
	'homepageTautulliViewsAuth' => '1',
	'homepageTraktAuth' => '1',
	'homepageTraktEnabled' => false,
	'homepageTransmissionAuth' => '1',
	'homepageTransmissionEnabled' => false,
	'homepageUnifiAuth' => '1',
	'homepageUnifiEnabled' => true,
	'homepageUnifiRefresh' => '600000',
	'homepageUseCustomStreamNames' => false,
	'homepageWeatherAndAirAirQualityEnabled' => false,
	'homepageWeatherAndAirAuth' => '1',
	'homepageWeatherAndAirEnabled' => false,
	'homepageWeatherAndAirLatitude' => '',
	'homepageWeatherAndAirLongitude' => '',
	'homepageWeatherAndAirPollenEnabled' => false,
	'homepageWeatherAndAirRefresh' => '3600000',
	'homepageWeatherAndAirUnits' => 'imperial',
	'homepageWeatherAndAirWeatherEnabled' => false,
	'homepageWeatherAndAirWeatherHeader' => '',
	'homepageWeatherAndAirWeatherHeaderToggle' => false,
	'homepageqBittorrentAuth' => '1',
	'homepageqBittorrentEnabled' => true,
	'homepagerTorrentAuth' => '1',
	'homepagerTorrentEnabled' => false,
	'ignoreTFALocal' => false,
	'ignoredNewsIds' => array(

	),
	'installedPlugins' => '',
	'installedThemes' => 'Space Gray Theme:1.0.1',
	'jackettToken' => '0h6usb88ptfrk43lart9sm8xnp55b7yx',
	'jackettURL' => 'http://192.168.88.2:9117/',
	'jdownloaderCombine' => false,
	'jdownloaderURL' => '',
	'jellyfinSSOURL' => '',
	'jellyfinTabName' => '',
	'jellyfinTabURL' => '',
	'jellyfinToken' => '60e482b3fc8343a6857aad4c556d92e5',
	'jellyfinURL' => 'http://192.168.88.2:8096',
	'ldapBindPassword' => '',
	'ldapBindUsername' => '',
	'ldapSSL' => false,
	'ldapTLS' => false,
	'ldapType' => '1',
	'license' => 'personal',
	'lidarrSocksAuth' => '999',
	'lidarrSocksEnabled' => false,
	'lidarrToken' => '',
	'lidarrURL' => '',
	'localAddress' => '',
	'localIPFrom' => '',
	'localIPTo' => '',
	'lockScreen' => false,
	'lockoutMaxAuth' => '0',
	'lockoutMinAuth' => '1',
	'lockoutSystem' => false,
	'lockoutTimeout' => '60',
	'loginAttempts' => '5',
	'loginLockout' => '60000',
	'loginLogo' => 'plugins/images/organizr/logo-wide.png',
	'loginWallpaper' => '',
	'logo' => 'plugins/images/organizr/logo-wide.png',
	'mediaSearch' => false,
	'mediaSearchAuth' => '1',
	'mediaSearchType' => '',
	'minimalLoginScreen' => false,
	'monitorrCompact' => false,
	'monitorrHeader' => 'Monitorr',
	'monitorrHeaderToggle' => true,
	'monitorrURL' => '',
	'netdata1Chart' => 'easypiechart',
	'netdata1Colour' => 'CC22AA',
	'netdata1Data' => 'disk-avail',
	'netdata1Enabled' => true,
	'netdata1Size' => 'md',
	'netdata1Title' => 'Available data in Pool',
	'netdata1lg' => true,
	'netdata1md' => true,
	'netdata1sm' => true,
	'netdata2Chart' => 'gauge',
	'netdata2Colour' => '5054e6',
	'netdata2Data' => 'ram-used',
	'netdata2Enabled' => true,
	'netdata2Size' => 'md',
	'netdata2Title' => 'Available RAM',
	'netdata2lg' => true,
	'netdata2md' => true,
	'netdata2sm' => true,
	'netdata3Chart' => 'gauge',
	'netdata3Colour' => 'fe3912',
	'netdata3Data' => 'disk-write',
	'netdata3Enabled' => false,
	'netdata3Size' => 'md',
	'netdata3Title' => 'CPU usage',
	'netdata3lg' => true,
	'netdata3md' => true,
	'netdata3sm' => true,
	'netdata4Chart' => 'easypiechart',
	'netdata4Colour' => '46e302',
	'netdata4Data' => 'disk-read',
	'netdata4Enabled' => false,
	'netdata4Size' => 'lg',
	'netdata4Title' => '',
	'netdata4lg' => true,
	'netdata4md' => true,
	'netdata4sm' => true,
	'netdata5Chart' => 'easypiechart',
	'netdata5Colour' => 'fe3912',
	'netdata5Data' => 'disk-read',
	'netdata5Enabled' => false,
	'netdata5Size' => 'lg',
	'netdata5Title' => '',
	'netdata5lg' => true,
	'netdata5md' => true,
	'netdata5sm' => true,
	'netdata6Chart' => 'easypiechart',
	'netdata6Colour' => 'fe3912',
	'netdata6Data' => 'disk-read',
	'netdata6Enabled' => false,
	'netdata6Size' => 'lg',
	'netdata6Title' => '',
	'netdata6lg' => true,
	'netdata6md' => true,
	'netdata6sm' => true,
	'netdata7Chart' => 'easypiechart',
	'netdata7Colour' => 'fe3912',
	'netdata7Data' => 'disk-read',
	'netdata7Enabled' => false,
	'netdata7Size' => 'lg',
	'netdata7Title' => '',
	'netdata7lg' => true,
	'netdata7md' => true,
	'netdata7sm' => true,
	'netdataCustom' => '{

	}',
	'netdataURL' => '192.168.88.2:19999',
	'notificationBackbone' => 'izi',
	'notificationPosition' => 'br',
	'nzbgetCombine' => false,
	'nzbgetPassword' => '',
	'nzbgetURL' => '',
	'nzbgetUsername' => '',
	'octoprintHeader' => 'Octoprint',
	'octoprintHeaderToggle' => true,
	'octoprintToken' => '',
	'octoprintURL' => '',
	'ombiAlias' => false,
	'ombiDefaultFilterApproved' => true,
	'ombiDefaultFilterAvailable' => true,
	'ombiDefaultFilterDenied' => true,
	'ombiDefaultFilterUnapproved' => true,
	'ombiDefaultFilterUnavailable' => true,
	'ombiFallbackPassword' => '',
	'ombiFallbackUser' => '',
	'ombiLimit' => '50',
	'ombiLimitUser' => false,
	'ombiRefresh' => '600000',
	'ombiToken' => '',
	'ombiTvDefault' => 'all',
	'ombiURL' => '',
	'organizrAPI' => 'ogj5eh6kssjqr7nd27qo',
	'organizrDocsMenuLink' => false,
	'organizrFeatureRequestLink' => false,
	'organizrHash' => '34rji30rij30r9j3r093r09jf3039r',
	'organizrSignoutMenuLink' => false,
	'organizrSupportMenuLink' => false,
	'otherPingRefresh' => '600000',
	'overseerrFallbackPassword' => '',
	'overseerrFallbackUser' => '',
	'overseerrToken' => '',
	'overseerrURL' => '',
	'performanceDisableIconDropdown' => true,
	'performanceDisableImageDropdown' => true,
	'piholeHeaderToggle' => true,
	'piholeURL' => '',
	'pingAuth' => '1',
	'pingAuthMessage' => '1',
	'pingAuthMs' => '1',
	'pingMs' => false,
	'pingOfflineSound' => 'plugins/sounds/default/open-ended.mp3',
	'pingOnlineSound' => 'plugins/sounds/default/awareness.mp3',
	'plexAdmin' => '',
	'plexID' => '',
	'plexStrictFriends' => true,
	'plexTabName' => '',
	'plexTabURL' => '',
	'plexToken' => '',
	'plexURL' => '',
	'plexoAuth' => false,
	'qBittorrentApiVersion' => '2',
	'qBittorrentCombine' => false,
	'qBittorrentDisableCertCheck' => true,
	'qBittorrentHideCompleted' => false,
	'qBittorrentHideSeeding' => false,
	'qBittorrentPassword' => 'x0IK0QQdXgV0r7M9emwF+Q==',
	'qBittorrentReverseSorting' => false,
	'qBittorrentSortOrder' => 'eta',
	'qBittorrentURL' => 'http://192.168.88.2:9090',
	'qBittorrentUsername' => 'Asterix',
	'rTorrentCombine' => false,
	'rTorrentDisableCertCheck' => false,
	'rTorrentHideCompleted' => false,
	'rTorrentHideSeeding' => false,
	'rTorrentLimit' => '200',
	'rTorrentPassword' => '',
	'rTorrentReverseSorting' => false,
	'rTorrentSortOrder' => 'datea',
	'rTorrentURL' => '',
	'rTorrentURLOverride' => '',
	'rTorrentUsername' => '',
	'radarrCinemaRelease' => false,
	'radarrDigitalRelease' => false,
	'radarrPhysicalRelease' => true,
	'radarrSocksAuth' => '999',
	'radarrSocksEnabled' => false,
	'radarrToken' => '1c0e90cb01564d97ad3363bab136a3ab',
	'radarrURL' => '192.168.88.2:7878',
	'radarrUnmonitored' => false,
	'registrationPassword' => '4545',
	'rememberMe' => true,
	'rememberMeDays' => '7',
	'sabnzbdCombine' => false,
	'sabnzbdToken' => '',
	'sabnzbdURL' => '',
	'sandbox' => 'allow-presentation,allow-forms,allow-same-origin,allow-pointer-lock,allow-scripts,allow-popups,allow-modals,allow-top-navigation',
	'selfSignedCert' => '',
	'sickrageToken' => '',
	'sickrageURL' => '',
	'sidebarColor' => '',
	'sidebarTextColor' => '',
	'sonarrSocksAuth' => '999',
	'sonarrSocksEnabled' => false,
	'sonarrToken' => 'b21347fdb30d4138bccc2a76fcc634be',
	'sonarrURL' => 'http://192.168.88.2:8989/',
	'sonarrUnmonitored' => true,
	'speedtestHeader' => 'Speedtest',
	'speedtestHeaderToggle' => true,
	'speedtestURL' => '',
	'ssoJellyfin' => false,
	'ssoOmbi' => false,
	'ssoOverseerr' => false,
	'ssoPlex' => false,
	'ssoTautulli' => false,
	'statusSounds' => false,
	'style' => 'dark',
	'tautulliApikey' => '',
	'tautulliFriendlyName' => true,
	'tautulliHeader' => 'Tautulli',
	'tautulliHeaderToggle' => true,
	'tautulliLibraries' => true,
	'tautulliPopularMovies' => true,
	'tautulliPopularTV' => true,
	'tautulliTopMovies' => true,
	'tautulliTopPlatforms' => true,
	'tautulliTopTV' => true,
	'tautulliTopUsers' => true,
	'tautulliURL' => '',
	'theme' => 'Organizr',
	'themeInstalled' => '',
	'themeVersion' => '',
	'title' => 'Organizr V2',
	'traefikAuthEnable' => false,
	'traktAccessToken' => '',
	'traktAccessTokenExpires' => '',
	'traktClientId' => '',
	'traktClientSecret' => '',
	'traktRefreshToken' => '',
	'transmissionCombine' => false,
	'transmissionDisableCertCheck' => false,
	'transmissionHideCompleted' => false,
	'transmissionHideSeeding' => false,
	'transmissionPassword' => 'x0IK0QQdXgV0r7M9emwF+Q==',
	'transmissionURL' => 'http://192.168.88.2:9091/transmission',
	'transmissionUsername' => 'admin',
	'unifiCookie' => '',
	'unifiPassword' => 'ysmWLVhWnq5XjIixqiIbJA==',
	'unifiSiteName' => 'default',
	'unifiURL' => 'https://192.168.88.2:8443',
	'unifiUsername' => 'Asterix',
	'unsortedTabs' => 'top',
	'useLogo' => false,
	'useLogoLogin' => false,
	'uuid' => 'ae81d8a5-6d14-4f18-ba52-38e115f5095d',
	'wanDomain' => '',
	'youtubeAPI' => ''
);